public/app/features/alerting/unified/components/rules/central-state-history/EventListSceneObject.tsx

```diff
@@ -517,13 +521,16 @@ export function HistoryEventsListObjectRenderer({ model }: SceneComponentProps<H
     const newFilterToAdd = `${key}=${value}`;
     if (type === 'stateTo') {
       stateToFilterVariable.changeValueTo(value);
+      trackUseCentralHistoryFilterByClicking({ type: 'stateTo', key, value });
```

Comment:
Couldn't we just do `trackUseCentralHistoryFilterByClicking({ type, key, value })` once at the top?

Reply:
💯